src/app/modules/products/components/featured-product-card/featured-product-card.component.html

```diff
@@ -1,16 +1,80 @@
-<div class="featured-product-card">
-    <div class="featured-product-card-image-container">
-        <img class="featured-product-card-image" src="https://images.unsplash.com/photo-1475855581690-80accde3ae2b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80" alt="">
+<ng-container *ngIf="product != null; else showLoader">
```

Comment:
I dont like how there are two templates and two ngIfs (one in product-page and one in the card components

Feels like we are checking the same condition twice.

Let's move all the html for the loaders in their own components and then add them in the product-page:

`
<app-grid-list [itemColumnSpan]=2 [gridGap]=4>
        <ng-container *ngIf="featuredProducts$ | async as featuredProducts; else featuredProductsLoading">
            <app-featured-product-card *ngFor="let product of featuredProducts" [product]="product"></app-featured-product-card>
        </ng-container>
        <ng-template #featuredProductsLoading>
            <app-featured-product-card-loader></app-featured-product-card-loader>
        </ng-template>
    </app-grid-list>
`